modules/ROOT/pages/managing-asset-deletion.adoc

```diff
@@ -0,0 +1,16 @@
+= Managing Asset Deletion
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Control how assets are deleted in Exchange from *Settings*. Admins can choose how asset owners can delete asset versions in Exchange. They can enable or disable the ability to permanently delete an asset to reuse the GAV (group ID, asset ID, and version) for a new asset. It doesn't matter when the asset version was created. When enabled, asset owners can see an option to delete an asset and allow the GAV to be reused. When disabled, the option to delete the asset but not the reuse the GAV is available.
```

Comment:
Maybe something like this?
`When disabled, the GAV will not be reusable after deleting an asset.`